src/internal/Observable.ts

```diff
@@ -251,12 +251,64 @@ export class Observable<T> implements Subscribable<T> {
   }
 
   /**
-   * @method forEach
+   * Used as a NON-CANCELLABLE means of subscribing to an observable, for use with
```

Comment:
Is it worth mentioning [rxjs-for-await](https://www.npmjs.com/package/rxjs-for-await)? I'm 50/50

Reply:
I'm not sure. Debatable. I'll leave it for now.